server/api/v1/api.py

```diff
@@ -0,0 +1,10 @@
+from fastapi import APIRouter
+
+from server.api.v1.endpoints import buoys, coastlines
+from server.core.constants import BUOYS_PATH, COASTLINES_PATH, PathTags
+
+api_router = APIRouter()
+api_router.include_router(buoys.router, prefix=BUOYS_PATH, tags=[PathTags.BUOYS])
+api_router.include_router(
+    coastlines.router, prefix=COASTLINES_PATH, tags=[PathTags.COASTLINES]
+)
```

Comment:
Why did you create a separate api.py file for all the routes if they're all going to be included in the main.py file anyways? I don't really see how it'll create tech debt if there's going to be just multiple lines of routers in main.py which can be easily traced back to endpoints folder. It's what we already had except with extra steps.